templates/public/donate.html

```diff
@@ -41,25 +41,25 @@ <h3>Commercial sponsors and contributions</h3>
     Kartenzia.de is a new Start-Up based in Germany and specializes in
     eco-friendly invitations like <a href="http://www.kartenzia.de/weihnachtskarten/">Weihnachten</a>.</p>
 
-    <a href="https://www.kartenzia.de/"><img width="292" height="74" src="{% static "kartenzia_button.png" %}" class="sponsor-btn-kartenzia" /></a>
+    <a href="https://www.kartenzia.de/"><img src="{% static "kartenzia_button.png" %}" class="sponsor-btn-kartenzia" /></a>
 
     <p>We would like to thank <a href="https://www.privateinternetaccess.com/"
     title="Private Internet Access">Private Internet Access</a> for sponsoring
     dedicated servers across the globe. Private Internet Access is the leading
     VPN Service provider specializing in secure, encrypted VPN tunnels which
     create several layers of privacy and security providing users safety on the
-    internet</p>.
+    internet.</p>
 
     <a href="https://www.privateinternetaccess.com/" title="Private Internet Access">
-    <img width="496" height="76" src="{% static "pia_logo.png" %}"
+    <img src="{% static "pia_logo.png" %}"
     class="sponsor-btn-pia" title="" alt="Private Internet Access logo"/></a>
 
     <p>We would also like to thank <a href="https://icons8.com/" title="Icons8">
     Icons8</a> for their yearly monetary donation. Icons8 is a team of talented
     designers and engineers working on fulfilling all your icons needs.</p>
 
     <a href="https://icons8.com/" title="Icons8">
-    <img width="200" height="200" src="{% static "icons8_logo.png" %}"
+    <img src="{% static "icons8_logo.png" %}"
```

Comment:
All style changes, please make a separate commit / PR for them

Reply:
The specified width and height on these images breaks on small screen widths.